pageserver/src/context.rs

```diff
@@ -98,6 +100,8 @@ pub struct RequestContext {
     access_stats_behavior: AccessStatsBehavior,
     page_content_kind: PageContentKind,
     pub micros_spent_throttled: optional_counter::MicroSecondsCounterU32,
+    /// Total number of kilobytes of layer files processed in this request.
+    pub vectored_access_delta_file_size_kb: AtomicUsize,
```

Comment:
Is this used anywhere?

Reply:
Oh it’s from my other pull request, will remove it.

Reply:
fixed in https://github.com/neondatabase/neon/pull/7623/commits/13977eae249f82ec38cdaaf2573adfabac971003